proxy/src/main/java/com/wavefront/agent/PushAgent.java

```diff
@@ -1024,6 +1027,10 @@ protected void processConfiguration(AgentConfiguration config) {
         entityProps.getGlobalProperties().
             setHistogramStorageAccuracy(config.getHistogramStorageAccuracy().shortValue());
       }
+      entityProps.getGlobalProperties().setTraceSamplingRate(config.getSpanSamplingRate());
+      rateSampler.setSamplingRate(entityProps.getGlobalProperties().getTraceSamplingRate());
+      logger.fine("Proxy trace span sampling rate set to " +
```

Comment:
Nit: maybe do a logger.info instead but only when the value actually changes? lgtm otherwise.

Reply:
done.